elixir.html.markdown

```diff
@@ -411,7 +412,8 @@ self() #=> #PID<0.27.0>
 
 * [Getting started guide](http://elixir-lang.org/getting_started/1.html) from [elixir webpage](http://elixir-lang.org)
 * [Elixir Documentation](http://elixir-lang.org/docs/master/)
-* ["Programming Elixir"](https://pragprog.com/book/elixir/programming-elixir) by Dave Thomas
+* [Programming Elixir](https://pragprog.com/book/elixir/programming-elixir) by Dave Thomas
```

Comment:
Why did you remove double quotes? This is a book and it should be rendered as `"Some Book" by Author` rather than `Some Book by Author`.